code/protected/Dao/GrootsledgerDao.php

```diff
@@ -13,6 +13,7 @@ public static function savePayment($retailer, $paid_amount){
         	$retailer->collection_fulfilled = false; 
         }
         $retailer->updated_at = date("Y-m-d H:i:s");
```

Comment:
$retailer->updated_at should not be at code level, it should be auto updated by db.